src/containers/BooksList/books-list.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import './books-list.css';
+import CategoryFilter from '../../components/categoryFilter';
+import Book from '../../components/Book';
+import { removeBook } from '../../actions';
+
+const BooksList = ({ books, removeBook, filter }) => {
+  const filteredBooks = !filter ? books : books.filter(
+    book => book.category === filter,
+  );
+
+  return (
+    <div className="booksList">
+      <CategoryFilter />
+      <table>
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Title</th>
+            <th>Category</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          { filteredBooks.map(book => <Book key={book.id} book={book} removeBook={removeBook} />) }
+        </tbody>
+      </table>
+    </div>
+  );
+};
+
```

Comment:
i think you forgot one more task :
implement `handleFilterChange` and pass it to `<CategoryFilter>`
[[see requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/HandelChangeFilter.png)]